OODebug/src/debugger/JavaDebugger.cpp

```diff
@@ -291,7 +294,8 @@ Interaction::CommandResult* JavaDebugger::probe(OOVisualization::VStatementItemL
 
 	addBreakpointListener(breakpointId, createListenerFor(observer, plotOverlay));
 
-	qDebug() << "Added probe" << breakpointId;
+	// TODO optimally we would represent this somehow in the user interface (maybe in the plot overlay).
```

Comment:
Yes, getting the log to be displayed in a place other than the console would be nice.